examples/next/next.config.js

```diff
@@ -3,6 +3,11 @@ const { patchWebpackConfig } = require('next-global-css');
 
 module.exports = {
   reactStrictMode: true,
+  /**
+   * Setting trailingSlash to true to resolve known bug with federated sign in redirect + next.js
+   * https://github.com/aws-amplify/amplify-cli/issues/7359#issuecomment-812821315
+   */
+  trailingSlash: true,
```

Comment:
See https://github.com/aws-amplify/amplify-cli/issues/7359 for details, but there is a known problem with redirect urls on next.js + federatedSignIn. 

This is the easiest workaround without modifying amplify configs, so went with this approach.